packages/salesforce-adapter/src/client/jsforce.ts

```diff
@@ -102,10 +102,17 @@ export interface RunTestFailure {
   time: number
 }
 
+export interface CodeCoverageWarning {
+  id: string
+  message: string
+  name?: string
+  namespace?: string
+}
+
 export interface RunTestsResult {
   apexLogId?: string
   codeCoverage?: ArrayOrSingle<object> // CodeCoverageResult[]
-  codeCoverageWarnings?: ArrayOrSingle<object> // CodeCoverageWarning[]
+  codeCoverageWarnings?: ArrayOrSingle<CodeCoverageWarning> // CodeCoverageWarnings[]
```

Comment:
since you fixed the type, there is no need to have the comment anymore
```suggestion
  codeCoverageWarnings?: ArrayOrSingle<CodeCoverageWarning>
```

Reply:
v